types/custom-functions-runtime/index.d.ts

```diff
@@ -75,6 +75,15 @@ declare namespace CustomFunctions {
          * [Api set: CustomFunctionsRuntime 1.3]
          */
         parameterAddresses?: string[];
+
+        /**
+         * This flag determines whether the function is being invoked as part of Formula Value Preview execution or not.
+         * If the function is being invoked from Formula Value Preview, the flag is set to true; otherwise, it is set to false.
```

Comment:
Is this readonly? Is the default value false? Why `string[]` instead of just `string`? 

If I understand correctly, this can be used to set the true/false value, so we should use active voice (per Microsoft writing style guide), maybe like this:  
```suggestion
         * To invoke this function in the formula value preview, set to `true`. The default value is `false`.
```

Reply:
Thanks for reviewing this. This is not a flag that the developer sets -- rather it is a flag they use to detect whether the function was invoked by formula value preview or not. Using this flag, if the function was invoked by value preview, they can detect it and choose to exhibit some other behaviour rather than invoking some costly API. So it is not active voice but rather a readonly with no "default" value.